### ISSUE TITLE
Ping metrics are consistently the same value

### ISSUE THREAD

**Issue by kennedymeadows** [kentik/ktranslate#657]
I'm currently collecting pings on ~700 devices in our network. Some of these devices have ping metrics collected in addition to snmp and others are ping only. We've noticed that across the board that the values collected seem to get "stuck". What is returned every metric collection window (5 minutes in our case) will be the same value for hours on end (although not a consistent number of hours).

**Config with snmp and ping**:
```
trap:
    listen: 0.0.0.0:1621
    community: ....
    version: ""
    transport: ""
    v3_config: null
    trap_only: false
discovery:
    cidrs:
        - ...
    ignore_list:
        - ...
    debug: false
    ports:
        - 161
    default_communities:
        - ...
    use_snmp_v1: false
    default_v3: null
    add_devices: true
    add_mibs: true
    threads: 4
    replace_devices: true
    no_dedup_engine_id: true
    check_all_ips: true
    kentik: null
global:
    poll_time_sec: 300
    drop_if_outside_poll: false
    mib_profile_dir: /etc/ktranslate/profiles
    mibs_db: ""
    mibs_enabled:
        - BGP4-MIB
...
        - UPS-MIB
    timeout_ms: 3000
    retries: 0
    global_v3: null
    response_time: true
    purge_devices_after_num: 0
    no_device_hardcoded_oids: false
    user_tags: {}
    match_attributes: {}
    providers: {}
    jitter_time_sec: 0
```

**Config for ping only:**
```
  example_dev:
    device_name: example_dev
    device_ip: ...
    provider: kentik-ping
    user_tags:
      location: office
    ping_only: true
trap:
  listen: 0.0.0.0:1625
  community: ...
  version: ''
  transport: ''
  v3_config:
  trap_only: false
discovery:
  cidrs: []
  ignore_list: []
  debug: false
  ports:
    - 161
  default_communities:
    - ...
  use_snmp_v1: false
  default_v3:
  add_devices: true
  add_mibs: true
  threads: 4
  replace_devices: true
  no_dedup_engine_id: true
  check_all_ips: true
  kentik:
global:
  poll_time_sec: 300
  drop_if_outside_poll: false
  mib_profile_dir: /etc/ktranslate/profiles
  mibs_db: ''
  mibs_enabled: []
  timeout_ms: 1000
  retries: 0
  global_v3:
  response_time: true
  purge_devices_after_num: 0
  no_device_hardcoded_oids: false
  user_tags: {}
  match_attributes: {}
  providers: {}
  jitter_time_sec: 0
```

**The startup command** I use to begin each ktranslate process, taken from the command used by the docker container - we run this in a non-containerized environment:
```
ktranslate -listen off -mapping "${original_dir}config.json" \
            -geo "${original_dir}GeoLite2-Country.mmdb" \
            -udrs "${original_dir}udr.csv" \
            -api_devices "${original_dir}devices.json" \
            -asn "${original_dir}GeoLite2-ASN.mmdb" \
            -log_level warn -snmp "${file}" \
            -nr_account_id=399786 -metrics=jchf -tee_logs=true \
            -service_name="${service_name}" nr1.snmp > "/var/log/ktranslate-${service_name}.log"
```

I've checked through issues, github, and documentation and seen no mention so I'm hoping that this is a configuration issue I've missed.

![image](https://github.com/kentik/ktranslate/assets/62924402/b28dc70d-dda0-4a04-8d19-0f87e84a7b60)

**Comment by kennedymeadows** [comment]
I wanted to provide some further information, this is the data from one switch, the Max/Min Rtt values do not change very often but the averages do show slight differences between polling windows 
![image](https://github.com/kentik/ktranslate/assets/62924402/2e4dddd2-8a73-4dd1-bcc7-a91b7e81e60c)

**Comment by i3149** [comment]
This is happening because the stats object is being kept across upload events. So the min/max/avg isn't changing because there's a long history and it isn't doing a moving average very well. This seems non optimal. Let me move to a new system where min/max/avg is just for probes sent in the interval between data uploads.

**Comment by i3149** [comment]
Sorry for the delay here. Can you pull `kentik/ktranslate:staging` image and see if this fixes your issue?

**Comment by kennedymeadows** [comment]
Thanks @i3149 - tested the staging branch on a few of our devices and yes this fixed the issue. As you can see from the screenshot our regular collector (doing collections every 5 minutes) which is using ktranslate:v2 is stuck around 730ms RTT. Once the collector gets turned on using the staging branch the Max RTT is clearly fluctuating every polling window and is a realistic number. Will keep an eye out for the next main release so we can update our collectors.
![image](https://github.com/kentik/ktranslate/assets/62924402/06826f9c-be71-46a2-8b57-a96b40153d5c)

**Comment by i3149** [comment]
We went ahead and released [v2.1.25](https://github.com/kentik/ktranslate/releases/tag/v2.1.25) with this fix included. Docker images for `kentik/ktranslate:v2.1.25` now.
